.env.sample.b2c

```diff
@@ -1,3 +1,6 @@
+# Note: If you are using Azure App Service, go to your app's Configuration,
+# and then set the following values into your app's "Application settings".
```

Comment:
This message is misleading since FLASK_DEBUG is also in here, and they most definitely shouldn't set that to true. Can you put the comment above the registration secrets to make it clear it's not about FLASK_DEBUG? (Generally, it's common for env variables locally to be different in production, like for database connection strings, so they may add more env variables that would be local-only in future)

Alternatively, that comment can stay in the "Deployment" tutorial

Reply:
I understand your point. For the sake of discussion, why was FLASK_DEBUG=True acceptable before this PR? We certainly anticipated app developers to subsequently deploy this sample to their hosting system and eventually make proper changes in the final production system.

Currently this paragraph is put at the very top of this file for visibility. I was bitten by the forgetting to setup App Service's application settings, and the troubleshooting experience was like shooting in the dark (those stream log does not seem to help - more on this later). I foresee the majority of future customers who attempts this sample on App Service would run into the same issue. We want to help them. "Burying" this paragraph in the middle of a file won't catch their attention.

How about, we do it the other way around? Let's comment out the FLASK_DEBUG=True line by default. And optionally you can modify the quickstart docs to append the `--debug` command line parameter in the "flask run ...". If you would agree on this direction, I can send out a different PR to bump our Flask dependency version to 2.2+ (the `--debug` is only available since Flask 2.2+).

Reply:
Good idea! I didn't realize the FLASK_DEBUG risk as all of the other samples I've worked on clearly use .env for local-only variables, and set the env variables to different values for prod (like for database connections). This ones different since you want to connect locally to the actual API.

I sent PR for debug command: https://github.com/MicrosoftDocs/azure-docs-pr/pull/232094